apps/backend/app/api/routes/visualizations.py

```
@@ -0,0 +1,257 @@
+import numpy as np
+from datetime import datetime
+
+router = APIRouter()
+
+
+@router.get("/histogram/{dataset_id}/{column_name}")
+async def get_histogram(
+    dataset_id: str,
+    column_name: str,
+    num_bins: Optional[int] = 50,
+    current_user_id: str = Depends(get_current_user_id),
```

Ruff (0.14.4)

Unused function argument: `current_user_id`

(ARG001)